Makefile

```diff
@@ -65,7 +65,7 @@ E2E_TARGET ?= ./test/e2e/...
 E2E_KIND_VERSION ?= kindest/node:v1.28.0
 
 # E2E_K8S_VERSIONS sets the list of k8s versions included in test-e2e-all
-E2E_K8S_VERSIONS ?= 1.24.15 1.25.11 1.26.6 1.27.3 1.28.0
+E2E_K8S_VERSIONS ?= 1.26.6 1.27.3 1.28.0 1.29.0
```

Comment:
@kannon92 Can we update the Kubernetes version in other places like `ENVTEST_K8S_VERSION`?

Reply:
Its at 1.28.0 and I don't know if we should update that to 1.29.0 yet.

Reply:
let's use the latest patch version for 1.28

Reply:
updated. sorry for the lag.